ArmPkg/Drivers/SmmuDxe/IoMmu.h

```diff
@@ -23,6 +23,39 @@
 #define PAGE_TABLE_DESCRIPTOR       (0x1 << 1)
 #define PAGE_TABLE_READ_WRITE_FROM_IOMMU_ACCESS(IoMmuAccess)  (IoMmuAccess << 6)
 
+typedef UINT64 PAGE_TABLE_ENTRY;
+
+#define PAGE_TABLE_SIZE  EFI_PAGE_SIZE / sizeof(PAGE_TABLE_ENTRY)  // Number of entries in a page table

```

Comment:
```suggestion
#define PAGE_TABLE_SIZE  (EFI_PAGE_SIZE / sizeof(PAGE_TABLE_ENTRY))  // Number of entries in a page table
```

I find that math based C macros can be dangerous without () as that can cause unexpected order-of-operation issues if inlined with other math. Good practice to just always wrap them in ().

Reply:
done